covid19/src/components/App.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Link
+} from "react-router-dom";
+import About from './About.jsx';
+
+export default function App() {
+  return (
+    <Router>
+      <div>
+        <nav>
+          <ul>
+            <li>
+              <Link to="/summary">Summary</Link>
+            </li>
+            <li>
+              <Link to="/about">About</Link>
+            </li>
+          </ul>
+        </nav>
+
+        <Switch>
+          <Route path="/about">
+            <About />
+          </Route>
+          <Route path="/summary">
+            <p>default data...</p>
```

Comment:
Some notes:

- I noticed that you've copied the example from the document I've sent you. That is fine. it is actually working fine. But please make the summary as it's own component.